src/main/java/com/griddynamics/pift/SQLUtils.java

```diff
@@ -80,4 +82,57 @@ private static String getColumnName(Field field) {
         }
         return field.getName();
     }
+
+    public static String createQueryForSelectById(Class<?> type, Object id){
+        return "SELECT * FROM " +
+                ReflectionUtils.getTableName(type) +
+                " WHERE " +
+                getColumnName(getIdField(type)) +
+                " = " + convertObjectToString(id);
+    }
+
+    private static String convertObjectToString(Object obj){
+        if (obj instanceof String) {
+            return "'" + obj + "'";
+        }
+        return obj.toString();
+    }
+
+    private static Field getIdField(Class<?> type) {
+        return Arrays.stream(type.getDeclaredFields())
+                .filter(x -> x.isAnnotationPresent(Id.class))
+                .findFirst().orElseThrow(() -> new IllegalArgumentException("Exception in getIdField method"));
+    }
+
+    /**
+     * Gets id field from object reference field.
+     *
+     * @param field that references another object.
+     * @return id field
+     */
+    private static Field getIdField(Object entity, Field field) {
+        return Arrays.stream(ReflectionUtils.getFieldValue(field, entity).getClass()
+                        .getDeclaredFields()).filter(x -> x.isAnnotationPresent(Id.class))
+                .findFirst().orElseThrow(() -> new IllegalArgumentException("Exception in getIdField method"));
+    }
+
+    private static String getQueryCondition(Object entity) {
+        StringBuilder res = new StringBuilder();
+        getNotNullColumnFields(entity).forEach(field -> {
+            if (!res.isEmpty()) {
+                res.append(" AND ");
+            }
+            res.append(field.getName());
+            res.append(" = ").append(SQLUtils.readField(field, entity));
+        });
+        return res.toString();
+    }
+
+    private static List<Field> getNotNullColumnFields(Object entity) {
+        return Arrays.stream(entity.getClass().getDeclaredFields())
+                .filter(field -> !field.isAnnotationPresent(JoinColumn.class))
```

Comment:
Need to create set of annotations (JoinColumn, Id, Version) and  in the separate method check that these annotations are not present